### PR TITLE
Add OLS-4 blog post

### PR DESCRIPTION
**Pull request by bebatut**
and fix some country names.

I just did not add keywords about the projects (I did have them as last time in the project sheet. OpenReview issue?)

### CODE REVIEW

File: _posts/2021-09-07-ols-4-annoucement.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+{% assign p-mentors = all-mentors | remove_first: ', ' | split: ", " | uniq | sort %}
+{% assign m-countries = all-m-countries | remove_first: ', ' | split: ", " | uniq | sort %}
+
+We are thrilled to announce that [{{ p-participants | size }} members](/{{ cohort }}/projects-participants/#participants), who are the project leads of [{{ projects | size }} diverse projects](/{{ cohort }}/projects-participants/#projects), have joined the fourth cohort of the Open Life Science mentoring program - OLS-4!
+
+### Meet our mentees!
+
+The mentees joining this program are {{ p-participants | join: ', ' }}. These individuals are based in {{ p-countries | size }} countries ({{ p-countries | join: ', ' }}) where they will be leading their respective projects. 
+
+Topics for their projects include {{ keywords | join: ', ' }}.
```
Comment (bebatut) [inline _posts/2021-09-07-ols-4-annoucement.md:92]: we need to add some keywords there as I did have them in the project description sheet

Comment (malvikasharan) [reply _posts/2021-09-07-ols-4-annoucement.md:92]: added some keywords.

File: _posts/2021-09-07-ols-4-annoucement.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    {% for k in project.keywords %}
+        {% capture all-keywords %}{{ all-keywords }}, {{ k }}{% endcapture %}
+    {% endfor %}
+{% endfor %}
+
+<!-- transform into lists -->
+{% assign p-participants = all-participants | remove_first: ', ' | split: ", " | uniq | sort %}
+{% assign p-countries = all-p-countries | remove_first: ', ' | split: ", " | uniq | sort %}
+{% assign p-mentors = all-mentors | remove_first: ', ' | split: ", " | uniq | sort %}
+{% assign m-countries = all-m-countries | remove_first: ', ' | split: ", " | uniq | sort %}
```
Comment (bebatut) [inline _posts/2021-09-07-ols-4-annoucement.md:84]:
```suggestion
{% assign keywords = all-keywords | remove_first: ', ' | split: ", " | uniq | sort %}
```

File: _posts/2021-09-07-ols-4-annoucement.md (hunk 1 above)
Comment (bebatut) [reply _posts/2021-09-07-ols-4-annoucement.md:92]: Thanks. I fixed sth above that should make them displayed

**Comment by malvikasharan** [comment]
This looks good @bebatut 🎉

**Pull request merged**